Libraries/Components/View/ViewStylePropTypes.js

```diff
@@ -33,6 +33,14 @@ var ViewStylePropTypes = {
   borderTopRightRadius: ReactPropTypes.number,
   borderBottomLeftRadius: ReactPropTypes.number,
   borderBottomRightRadius: ReactPropTypes.number,
+  borderTopLeftVerticalRadius: ReactPropTypes.number,
+  borderTopLeftHorizontalRadius: ReactPropTypes.number,
+  borderTopRightVerticalRadius: ReactPropTypes.number,
+  borderTopRightHorizontalRadius: ReactPropTypes.number,
+  borderBottomLeftVerticalRadius: ReactPropTypes.number,
+  borderBottomLeftHorizontalRadius: ReactPropTypes.number,
+  borderBottomRightVerticalRadius: ReactPropTypes.number,
+  borderBottomRightHorizontalRadius: ReactPropTypes.number,
```

Comment:
iOS doesn't seem to have equivalent properties to vary the horizontal and vertical radii at a corner, at least not at the moment - so these just won't do anything yet on iOS. Also, on iOS there isn't properties to set the corner radii of an image at the moment. With this change it would be possible to do this on Android so I've also updated ImageStylePropTypes in my latest commit.